tests/topotests/bfd-topo1/test_bfd_topo1.py

```diff
@@ -45,6 +45,9 @@
 # Required to instantiate the topology builder class.
 from mininet.topo import Topo
 
+pytestmark = [
+    pytest.mark.bfdd
```

Comment:
isn't bgp used too?

Reply:
again, I plan to come around for a bgp fixup all in one pass.  I tried to mark the really obvious to me when I was in there for bfdd.